src/test/resources/config/application-integration-test.yml

```diff
@@ -0,0 +1,12 @@
+workspace.integration-test:
+  data-repo-instance-names:
+    dev: terra
```

Comment:
I expected the value here to be `terra-dev, unless this is a welcome change?

Reply:
Yeah, there will be some other PR's that land before this, the goal is that each environment's data repo instance is just called `terra`.